packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -158,100 +153,103 @@ export class DhisService extends Service {
     return events;
   };
 
-  fetchEventAnalyticsForPrograms = async (api, programCodes, query) => {
-    const allHeaders = [];
```

Comment:
We used to merge first then translate, so we had to update the width and height to be correct (the translator relies on it - probs). After these changes, we translate first then merge, so we don't need this logic anymore